src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -158,7 +158,10 @@ public Transaction GetTransactionById(uint256 trxid)
             Guard.NotNull(trxid, nameof(trxid));
 
             if (!this.TxIndex)
+            {
+                this.logger.LogTrace("(-)[NOT_INDEXED]:null");
```

Comment:
I would change this to `TX_INDEXING_DISABLED` or `TX_INDEX_OFF` then its immediately more clear.

Reply:
fixed